k8s/namespaces/rd/rd-location-ref-data-load/aat-00.yaml

```diff
@@ -6,6 +6,6 @@ metadata:
 spec:
   values:
     job:
-      schedule: "30 17 * * *"
+      schedule: "0/10 * * * *"
```

Comment:
changing it to:

_At every 10th minute from 0 through 59_?

Reply:
yes